sbin/attest-server

```diff
@@ -1,10 +1,19 @@
 #!/usr/bin/python3
+
 """
 Quote and Eventlog validating Attestation Server.
 
-This server accepts incoming HTTP POST requests from an attesting machine
-that contain a tar file of a TPM Endorsement Key, Attestation Key, Quote,
-PCR list, Eventlog and other details. It performs three actions:
+This is a python flask server implementing a single API end-point on /, which
+expects a POST request encoded in conventional form (multipart/form-data) that
+includes a field called "quote" containing an input tarball (a source filename
```

Comment:
Should we remove the tar file entirely and instead post the individual components as named form fields?